### PR TITLE
Fix parla cpu/multiload/numpy dependency cycle.

### PR DESCRIPTION
**Pull request by insertinterestingnamehere**

### CODE REVIEW

**Comment by insertinterestingnamehere** [comment]
Fixes https://github.com/ut-parla/Parla.py/issues/11.

**Comment by insertinterestingnamehere** [comment]
This patch appears to work, but I'm certainly still putting it through its paces.

File: parla/cpu.py [hunk 1]
```diff
@@ -1,66 +1,17 @@
 import logging
 from typing import Dict, Collection
 
-# FIXME: This load of numpy causes problems if numpy is multiloaded. So this breaks using VECs with parla tasks.
-#  Loading numpy locally works for some things, but not for the array._register_array_type call.
```
Comment (arthurp) [inline parla/cpu.py:5]: I think this is still an issue with this code. Why did you remove the FIXME.

**Review by arthurp — APPROVED** [review]
Fixes the dependency cycle. But not the VEC issue as importing `parla.cpu` outside multiload will still mess things up by importing numpy early.

Otherwise, LGTM.

**Comment by insertinterestingnamehere** [comment]
Right. I think the idiom we want to use for now goes like this:

```
from parla.multiload import multiload

with multiload():
    import numpy

from parla.cpu import cpu
```

So, things work, but multiload has to be imported first. I don't think this is ideal yet, but it'll get things going for the VEC demos again.

File: parla/cpu.py (hunk 1 above)
Comment (insertinterestingnamehere) [reply parla/cpu.py]: I put it back in. This is only partially fixed now.
